alphagov/paas-cf
# [#147829023] Verify MySQL TLS certificate for healthcheck

## What

Enable and fix TLS verify for the healtcheck app using the MySQL service.

## How to review

1. Deploy the branch
1. Check that this command:

       curl -k https://healthcheck.${DEPLOY_ENV}.dev.cloudpipelineapps.digital/db\?service\=mysql\&ssl\=true

    Returns:

       {
         "success": true
       }

## Who can review

Anyone
